app/frontend/wallet/cardano-trezor-crypto-provider.js

```diff
@@ -14,6 +12,30 @@ const CardanoTrezorCryptoProvider = (ADALITE_CONFIG, walletState) => {
     throw new Error(`Unsupported derivation scheme: ${state.derivationScheme.type}`)
   }
 
+  const deriveXpub = generateDeriveXpubFn(async (absDerivationPath) => {
+    const response = await TrezorConnect.cardanoGetPublicKey({
+      path: absDerivationPath,
+      showOnTrezor: false,
+    })
+    if (response.error || !response.success) {
+      return Promise.reject(response || 'operation failed')
+    }
+    return Buffer.from(response.payload.publicKey, 'hex')
+  }, state.derivationScheme)
+
+  function deriveHdNode(childIndex) {
+    throw new Error('This operation is not supported on TrezorCryptoProvider!')
+  }
+
+  function sign(message, absDerivationPath) {
+    throw new Error('Not supported')
+  }
+
+  async function verifyAddressOnDevice(address, addressToAbsPathMapper) {
```

Comment:
I don't think that crypto provider should be asked to verify *address*, wouldn't path be better?
Basically, `displayAddressForPath` or something like that